templates/amazon-linux/cloud-init.sh

```diff
@@ -397,6 +397,13 @@ KONG_CLUSTER_TELEMETRY_ENDPOINT="${kong_hybrid_conf.endpoint}:${kong_ports.telem
 
 KONG_PLUGINS="${kong_plugins}"
 
+%{ if vitals_endpoint != "" && kong_vitals_enabled == "on" }
```

Comment:
Should this be

`%{ if vitals_endpoint != null ... ` ?

The default value is null, but you're checking against an empty string

Reply:
The default for the Terraform variable is null but it checks for null and passes an empty string here -> https://github.com/dwp/terraform-aws-kong-gateway/pull/100/files#diff-677655844bc7745b8c9085215751afe5a73658cd5a3e99ca03948bd80a77468fR62-R66

This was because Terraform didn't seem to like passing a null `object({})` to a templatefile